library/src/main/res/anim/invalid.xml

```diff
@@ -0,0 +1,8 @@
+<set xmlns:android="http://schemas.android.com/apk/res/android">
+    <translate
+        android:fromXDelta="-5%"
+        android:toXDelta="5%"
+        android:repeatCount="5"
+        android:repeatMode="reverse"
+        android:duration="150"/>
```

Comment:
Maybe use some already defined in android - https://developer.android.com/reference/android/R.integer.html#config_shortAnimTime